src/styles/typography.scss

```diff
@@ -29,129 +9,81 @@ $p-empty-state-height: 16px;
 $h6-empty-state-height: 12px;
 $small-empty-state-height: 12px;
 
+@mixin base-font {
```

Comment:
looks like the usage of `base-font` always comes along with a `font` rule. What do you think about adding the font rule as an argument to `base-font`, so that future devs will never forget that this is required?

Reply:
Agreed, will be fixed